modules/oadp-plugins.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+[id="oadp-plugins_{context}"]
+= OADP plug-ins
+
+Velero uses plug-ins that are integrated with storage providers. These plug-ins support backup and snapshot operations.
+
+The OpenShift API for Data Protection (OADP) provides default OADP plug-ins that contain Velero plug-ins for specific storage locations. You can create link:https://velero.io/docs/v1.7/custom-plugins/[custom plug-ins] based on the default plug-ins.
+
+[NOTE]
+====
+An OADP plug-in, for example, `aws`, can contain multiple plug-ins.
+====
```

Comment:
How is this useful? I see how the table describes the default plugins. But why is it important for us to know how they're built? Is this important for creating your own plugins?

Reply:
I was concerned that the user might not realize that "aws" plugin = backup plugin + snapshot plugin. But I can take this out.

No, it's not needed for creating plugins.